src/content/administration/dashboard.mdx

```diff
@@ -177,12 +177,64 @@ From this page you can follow a simple step-by-step guide to get the major compo
 
 ## Registry Credentials
 
-This section in the admin view allows you to see the registry credentials that have been [configured through the helm chart](/self-hosted/administration/configuration/#privateregistry). You will see a row for each configured registry with the type (Static or AWS) and the registry hostname. The 2 types of credentials supported are:
+This section of the Admin dashboard allows you to manage credentials for private registries that you want to use in your Okteto instance. Those credentials are injected by Okteto in any operation (build, deploy, etc...) done via CLI or the UI, so developers don't need access to them. You only need to configure your credentials here, and all the developers will be able to interact with those registries while developing with Okteto. If the setting [`daemonset.configurePrivateRegistriesInNodes.enabled`](administration/configuration/#daemonset) is enabled, Okteto's daemonset will also configure these credentials in the nodes, so the cluster can pull images from those registries.
+
+For each configured configured registry, you will see a row with the type (Static or AWS) and the registry hostname. The 2 types of credentials supported are:
 - Static: To configure registries with static username and password like Docker Hub.
-- AWS: To configure [ECR registries](https://docs.aws.amazon.com/ecr/index.html) using Access Key and Access Secret. In this case, Okteto takes care of refreshing docker credentials every 4 hours using the key and secret configured.
+- AWS: To configure [ECR registries](https://docs.aws.amazon.com/ecr/index.html) using Access Key and Access Secret. In this case, Okteto takes care of refreshing docker credentials every 4 hours using the key and secret configured. The hostname of the registry has to end with `.amazonaws.com`.
 
 <p align="center"><Image src="/docs/administration/images/registry_credentials_view.png" alt="Registry credentials view" width="900" /></p>
 
-For each configured registry you can see the non-sensitive information like username or hostname. For sensitive data you can only see some of the last characters as a hint to verify the value.
+Apart of adding new credentials, this view also allows you to rotate those credentials or delete them if you don't needed them anymore. For sensitive data you only see some of the last characters as a hint to verify the value.
+
+<p align="center"><Image src="/docs/administration/images/registry_credentials_detail.png" alt="Registry credentials detail" width="700" /></p>
+
+> Be aware that any change you do in those credentials might take a few minutes to be propagated to all the components in the cluster.
+
+### Advanced configuration
```

Comment:
I would move this section to another doc. Maybe a doc in `Self-hosted > Configuration`? Or in the community forum?

Reply:
Maybe a section about CRDs in general. We should document the git catalog CRD in case someone prefers to define the catalog in code.
But I feel CRDs is too low level to be included in a doc that explains the Admin View

Reply:
I added it here because there was not specific section for private registries or CRDs. I feel that one specific for private registry might be a small without too many content. One about CRDs might make sense as we can talk about others

Reply:
@pchico83  Moved to its own page. Take a look when you have a chance and let me know what you think